mass_mailing_resend/readme/CONTRIBUTORS.rst

```diff
@@ -2,3 +2,5 @@
 
   * Pedro M. Baeza
   * Ernesto Tejeda
+* `Trobz <https://trobz.com>`_:
+    * nguyenhk <nguyenhk@trobz.com>
```

Comment:
Incorrect RST syntax. Please leave an empty line.

Reply:
PR has been updated. Thank you.